website/docs/reference/commands/clone.md

```diff
@@ -5,7 +5,7 @@ id: "clone"
 ---
 
 The `dbt clone` command clones selected nodes from the [specified state](/reference/node-selection/syntax#establishing-state) to the target schema(s). This command makes use of the `clone` materialization:
-- If your data platform supports zero-copy cloning of tables, and this model exists as a table in the source environment, dbt will create it in your target environment as a clone
+- If your data platform supports zero-copy cloning of tables(Snowflake, Databricks, or BigQuery), and this model exists as a table in the source environment, dbt will create it in your target environment as a clone
```

Comment:
```suggestion
- If your data platform supports zero-copy cloning of tables (Snowflake, Databricks, or BigQuery), and this model exists as a table in the source environment, dbt will create it in your target environment as a clone
```

Reply:
might be good to also add here:
- https://docs.getdbt.com/best-practices/clone-incremental-models#what-happens-when-one-of-the-modified-models-or-one-of-their-downstream-dependencies-is-an-incremental-model
- https://docs.getdbt.com/blog/to-defer-or-to-clone